src/assets/crubrand/generic/bourbon/addons/_prefixer.scss

```diff
@@ -3,6 +3,12 @@
 //************************************************************************//
 // Variables located in /settings/_prefixer.scss
 
+$prefix-for-webkit:    true !default;
+$prefix-for-mozilla:   true !default;
+$prefix-for-microsoft: true !default;
+$prefix-for-opera:     true !default;
+$prefix-for-spec:      true !default;
```

Comment:
I'm trying to find the right way to setup these variables.  I think they're supposed to be loaded from src/assets/crubrand/generic/bourbon/settings/_prefixer.scss.  However, I was noticing strange behavior in the layout of some elements, which I traced back to the prefixer mixin in this file, and explicitly declaring these variable defaults in this file fixed the issue.

On a related note, I'm getting deprecation warnings for this file as well, in the disable-prefix-for-all mixin:

![Screen Shot 2022-02-04 at 9 51 30 AM](https://user-images.githubusercontent.com/35531817/152549887-21d9860b-4540-4cc2-ba85-e7a9980d0a23.png)

It's complaining about this same list of variables, and about trying to assign them globally.  I can look into this deeper, but if anyone has any insight on it I would appreciate it.

Reply:
BTW these changes that I made to `_prefixer.scss` fixed the weird `box-sizing` issue I mention above.

Reply:
Are there adverse effects to removing the `@mixin disable-prefix-for-all() {` block? I'm guessing it was put there for a reason, but I was not part of this codebase when it was being written.

Reply:
It looks like it gets referenced here: https://github.com/CruGlobal/give-web/blob/master/src/assets/crubrand/generic/bourbon/css3/_keyframes.scss.  That file also seems to try to reference the same variables declared in https://github.com/CruGlobal/give-web/blob/master/src/assets/crubrand/generic/bourbon/settings/_prefixer.scss.

Reply:
If that's the case, I could try copying the default values there as well, but it would probably be better if they were defined in one place that both files can access.

Reply:
https://stackoverflow.com/questions/60422903/cannot-make-variables-globally-accessible-in-sass